magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/UnsafeEvaluateInvocationCheck.java

```diff
@@ -23,6 +23,6 @@ protected void walkPreMethodInvocation(final AstNode node) {
       return;
     }
 
-    this.addIssue(node, MESSAGE);
+    this.addIssue(node.getFirstChild(MagikGrammar.METHOD_NAME), MESSAGE);
```

Comment:
Please add method to the `MethodInvocationNodeHelper` to get the method name node. I'm sure there are other checks which can use this as well.

Also, please don't nest method calls into method calls. This reduces the readability. Better to assign it to a variable first, with a descriptive name.